rskj-core/src/test/java/co/rsk/peg/PegUtilsTest.java

```diff
@@ -61,6 +66,34 @@ void init() {
         activeFederation = createP2shErpFederation(federationMainNetConstants, activeFedSigners);
     }
 
+    @ParameterizedTest
+    @MethodSource("derivationHashAndRedeemScriptArgs")
+    void getFlyoverScriptPubKey_fromRealValues_shouldReturnSameRealOutputScript(Keccak256 flyoverDerivationHash, Script redeemScript) {
+        Script scriptPubKey = getFlyoverScriptPubKey(flyoverDerivationHash, redeemScript); // OP_HASH160 outputScript OP_EQUAL
+        byte[] outputScript = Hex.decode("18fc3b52a5b7d5277f41b9765719b45bfa427730");
+
+        assertArrayEquals(outputScript, scriptPubKey.getPubKeyHash());
+    }
+
+    @ParameterizedTest
+    @MethodSource("derivationHashAndRedeemScriptArgs")
+    void getFlyoverAddress_fromRealValues_shouldReturnSameRealAddress(Keccak256 flyoverDerivationHash, Script redeemScript) {
+        Address flyoverAddress = new Address(btcMainnetParams, "33y8JWrSe4byp3DKmy2Mkyykz2dzP8Lmvn");
```

Comment:
Check sonar warning, you can use Address.fromBase58 here

Reply:
thanks! updated